packages/vscode/src/plugins/nextjs/addImports.ts

```diff
@@ -0,0 +1,35 @@
+import { SourceFile } from 'ts-morph'
+import { NextFunctionName, NextFunctionType } from './constants'
+
+/**
+ * Adds Missing Next Imports depending function which have been found
```

Comment:
This comment is quite hard to understand imo.

Reply:
How about:
Adds the Required NextJS type import if they are missing

Reply:
How about: 

`Adds types to untyped next.js functions`

Or like it is advised to write declarative comments

`Add types to untyped next.js functions`

Reply:
@divyenduz That would be more applicable to `addTypesToPage.ts`. This will only add the imports to the file and only if they are missing 
```typescript
import { InferGetStaticPropsType } from "next";
```